config/initializers/paths.rb

```diff
@@ -3,26 +3,14 @@ module PathsInitializer
 
   class << self
     def perform_initialization(config)
-      if Rails.env == "test"
-        config.data_root = Rails.root.join('tmp','data')
-      else
-        config.data_root = Rails.root.join('data').sub(%r(/releases/\d+/), "/current/")
-      end
+      settings = Rails.env == "test" ? nil : Settings.git
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.